pkgs/development/tools/build-managers/build2/bpkg.nix

```diff
@@ -22,6 +22,15 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-3F4Pv8YX++cNa6aKhPM67mrt/5oE1IeoZUSmljHqBfI=";
   };
 
+  patches = [
+    # Patch git tests for git v2.38+
+    # Remove when bumping to v0.16.0 or greater
+    (fetchurl {
```

Comment:
why not fetchpatch?

Reply:
I've been told to use fetchurl in the past, when the .patch is usable as-is without any post-fetch fetchpatch operations

Reply:
Ok I might have been unclear on this, I found this guideline (https://github.com/NixOS/nixpkgs/issues/240141) which says to use fetchpatch for github patches, so I'll switch it over.